src/core/methods/signMessage/verifyMessage.ts

```diff
@@ -0,0 +1,35 @@
+import { Address, SignableMessage } from '@multiversx/sdk-core/out';
+import { getVerifier } from './getVerifier';
+
+export const verifyMessage = (signedMessage: string) => {
+  try {
+    const { message, address, signature } = JSON.parse(signedMessage);
+
+    const decodedMessage = Buffer.from(message.replace('0x', ''), 'hex');
+    const decodedSignature = Buffer.from(signature.replace('0x', ''), 'hex');
+    const bech32Address = new Address(address);
+    const signedM = new SignableMessage({
```

Comment:
we should migrate to the latest changes in sdk-core, so please use Message, instead of SignableMessage

Reply:
will do in v2 first